### DISCUSSION TITLE
modpack/kubejs/1.20.1/Introduction/Addon/LootJs/LootJs

### DISCUSSION THREAD

**Discussion by giscus** [PickAID/CrychicDoc#52] [Announcements]
# modpack/kubejs/1.20.1/Introduction/Addon/LootJs/LootJs

一个包含 Minecraft 开发文档的网站。

https://docs.mihono.cn/zh/modpack/kubejs/1.20.1/Introduction/Addon/LootJs/LootJs

**Comment by AnNingUI** [comment]
apply方法中通过context添加loot对玩家的挖掘没有用，只有像机械动力的动力钻头之类的机器破坏有用（悲
